examples/Crameri2012_case2/plot_max_surface.py

```diff
@@ -7,13 +10,13 @@
 import glob
 
 
-param_filename = 'param_1.5.3_2D.txt'
-filename_prefix = 'step_'
-interfaces_filename = 'interfaces_creep.txt'
+param_filename = "param_1.5.3_2D.txt"
```

Comment:
Not used, might be removed.

Reply:
@jamisonassuncao... you are right! I will remove it